website/docs/docs/dbt-versions/core-upgrade/04-upgrading-to-v1.11.md

```diff
@@ -0,0 +1,97 @@
+
+dbt Labs is committed to providing backward compatibility for all versions 1.x. Any behavior changes will be accompanied by a [behavior change flag](/reference/global-configs/behavior-changes#behavior-change-flags) to provide a migration window for existing projects. If you encounter an error upon upgrading, please let us know by [opening an issue](https://github.com/dbt-labs/dbt-core/issues/new).
+
+Starting in 2024, <Constant name="cloud" /> provides the functionality from new versions of <Constant name="core" /> via [release tracks](/docs/dbt-versions/cloud-release-tracks) with automatic upgrades. If you have selected the "Latest" release track in <Constant name="cloud" />, you already have access to all the features, fixes, and other functionality included in the latest <Constant name="core" /> version! If you have selected the "Compatible" release track, you will have access in the next monthly "Compatible" release after the <Constant name="core" /> v1.11 final release.
+
+We continue to recommend explicitly installing both `dbt-core` and `dbt-<youradapter>`. This may become required for a future version of dbt. For example:
+
+```sql
+python3 -m pip install dbt-core dbt-snowflake
+```
+
+## New and changed features and functionality
+
+New features and functionality available in <Constant name="core" /> v1.11
+
+### User-defined functions (UDFs) <Lifecycle status="beta" />
+
+dbt Core v1.11 introduces support for user-defined functions (UDFs), which enable you to define and register custom functions in your warehouse. Like macros, UDFs promote code reuse, but they run natively in the warehouse so you can reuse the same logic in tools outside dbt.
+
+Key features include:
+
+- **Define UDFs as first-class dbt resources**: Create UDF files in a `functions/` directory with corresponding YAML configuration.
+- **Native warehouse execution**: Create, update, and rename UDFs as part of DAG execution.
+- **DAG integration**: UDF files are built before models that reference them, ensuring proper dependency management.
+- **New `function()` macro**: Reference UDFs in your models using the `{{ function('function_name') }}` Jinja macro.
+- **Unit test support**: Validate models that use UDFs with unit tests. Make sure the functions exist by running `dbt build --select "+model_to_test" --empty` before running unit tests.
+- **List and select functions**: Manage UDFs by using `dbt list` and selection syntax like `dbt run --resource_type function` or `dbt run --select my_function`.
+
+Read more about UDFs, including prerequisites and how to define and use them in the UDF documentation.
+
+### Catalog improvements
+
+The catalog integration has been enhanced with new configuration options and validation:
+
+- `file_format` configuration is now supported in catalog integration.
+- Improved catalog and config validation for sources, models, and tests.
+- Better error handling and validation messages.
+
+### Enhanced `dbt ls` output
```

Comment:
what is this?

can we make this a "quick win" instead of calling it out as a feature